common/static/js/course_groups/cohorts.js

```diff
@@ -1,272 +0,0 @@
-// structure stolen from http://briancray.com/posts/javascript-module-pattern
-
-var CohortManager = (function ($) {
```

Comment:
I believe this code was from the first version of the cohort manager (in the instructor dashboard). It was written, but this file was left behind.

As far as I can tell, this is the place where profile.name was displayed when users moved between cohorts or were added to a cohort.